projects/igniteui-angular/src/lib/splitter/splitter.component.ts

```diff
@@ -151,6 +151,18 @@ export class IgxSplitterComponent implements AfterContentInit {
         this.sibling.size = siblingSize + 'px';
     }
 
+    /**
+     * This method performs the collapsing of the pane
+     * @param pane
+     */
+    public onCollapsing(pane: IgxSplitterPaneComponent) {
+        if (!pane) {
+            return;
+        }
+        this.pane = pane;
```

Comment:
I have to admit I am not too certain on the semantics but should we really reset the main pane? What happens to this.sibling then? They may become the same.

Reply:
Also, don't we need a public event for collapsing? Actually it should probably be something like paneToggle with an argument of the pane that changed its hidden state.